tests/main/views/test_audits.py

```diff
@@ -639,6 +643,8 @@ def test_should_get_audit_event(self):
             'acknowledged': False,
             'user': '0',
             'data': {'request': 'data'},
+            'objectType': 'Supplier',
+            'objectId': object_id,
```

Comment:
So suppliers is the weird one where `suppliers.id` != `supplier_id`.

Might this therefore be particularly confusing in that case? In most cases presumably the id is the id, but now we're exposing an identifier (for the first time) that otherwise, the API doesn't expose...

It's interesting that our tests seem to be covering the special-case model rather than any of the others... we probably should at least make sure the expected behaviour for other kinds of models is tested/documented somehow.

Maybe I am just confused :)

Reply:
I see what you mean about the weirdness with `supplier.id` and `supplier.supplier_id`, but I think this is exhibiting the expected behaviour. As I understand it we expect the `object_id` of an audit event to be that objects id, which it is here? The only reason I used a supplier as an object is because the `add_audit_event` method is set up to create a `supplier_update` audit event. I've only used the `supplier_id` to make sure I'm getting the correct supplier back when querying the `Supplier` model.
I could change it to use a different object, like a `User` maybe which would be more straight forward, but then the audit event and the object don't really match and it's a bit weird?

Reply:
Sure, I've no idea what the expected behaviour is for audit events, but it makes sense to be as you describe... 

What's in that `data` dict? I'm struggling to see how this serialization can have been any use before, if it didn't include some kind of identifier for the thing that changed!!

Reply:
It tends to be details of the audit event. For example, when updating the status of a framework it looks like this:
```
"data": {
    "update": {
        "status": "standstill"
    }
},
```
It was looking at exactly these events that prompted this PR. I was trying to work out when G-9 went from open to pending, and without the object id's of the frameworks, it was really difficult!

Other audit events have more details in them which make them more obvious, such as when updating a frameworks `frameworkAgreementDetails`. In that case the data blob includes stuff like `lotDescriptions` and `frameworkURL` so you can infer what the audit event was for more easily.